scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
```

Comment:
It's usually a good idea to have a `#!` line at the top of these to allow the interpreter to run the right script if it's invoked directly (e.g. `./run-bundled-codegen.sh`)

While Catalina will switch the default shell to `zsh`, it might actually be more defensive by just running this in `bash`, always?

```suggestion
#!/usr/bin/env bash
# Bail if errors occur
```

Reply:
added shebang